### PR TITLE
LPS-188399 Navigation menu missing tooltips, losing focus and not correct semantic labels

### PR DESCRIPTION
**Pull request by veroglez**
Fixes:

- https://liferay.atlassian.net/browse/LPS-188399
- https://liferay.atlassian.net/browse/LPS-188858

Waiting for:
https://github.com/liferay/clay/issues/5607

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -866,9 +866,9 @@ add-instance=Add Instance
 add-inventory=Add Inventory
 add-inventory-item=Add Inventory Item
 add-item=Add Item
-add-item-after-x=Add item after {0}.
+add-item-after-x=Add Item After {0}
 add-item-at-the-same-level=Add Item at the Same Level
-add-item-before-x=Add item before {0}.
+add-item-before-x=Add Item Before {0}
```
Comment (victorg1991) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:871]: Can we leave this as a sentence? or at least copy the translations that were already done

**Comment by victorg1991** [comment]
Also it seems that the focus is a bit off: 
Screenshot 2023-06-28 at 15 52 38

I believe we should set outline: none to that element

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (veroglez) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:871]: When it comes to displaying it in a tooltip, I think a sentence without a period is more correct, but I confirm it with Carol

**Comment by veroglez** [comment]
> Also it seems that the focus is a bit off: Screenshot 2023-06-28 at 15 52 38
> 
> I believe we should set outline: none to that element



I think it's better to replace padding by margin, otherwise this happens 😖 
Screen Shot 2023-06-28 at 8 49 12 PM

Result:
Screen Shot 2023-06-28 at 8 55 32 PM

**Comment by veroglez** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/12843

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**